README.md

```diff
@@ -78,15 +85,16 @@ Run `ssh deploy-api@fr.openfisca.org`
 
 This automatically runs the `fr.openfisca.org/api/deploy-latest.sh` script.
 
-### To serve a new version :
+### To serve a new version
+
 - on `openfisca-ops`, in a new branch, create a new repo with the config files
 - update the `fr.openfisca.org/api/deploy-latest.sh` symlink to point to the latest deploy script:
   - `cd fr.openfisca.org/api && rm deploy-latest.sh && ln -s ./vxx/deploy.sh deploy-latest.sh`
 - connect to the server **as `openfisca`**
```

Comment:
Why do I have to connect as openfisca and not with my sudoer login ?

Reply:
Because that user existence is to do so. Sudoers should IMHO only perform administrative tasks that goes beyond `openfisca`'s privileges.